### DISCUSSION TITLE
Feature editPost, documentation suggestion

### DISCUSSION THREAD

**Discussion by tiflor** [adityatelange/hugo-PaperMod#1010] [Ideas]
Hi ,

first of all thanks @adityatelange for this great theme.

While enabling it and got a little bit lost with the [editPost functionality](https://github.com/adityatelange/hugo-PaperMod/wiki/Features#edit-link-for-posts).
The documentation just stated some like ` URL: "https://github.com/<path_to_repo>/content" ` for the URL.
Adding my `path_to_repo` as `gitlabusername/reponame`is not doing the job to my understanding or do I have a misunderstanding here?

To my understanding the URL shall look like something similar to:

```
URL: "https://github.com/<gitlab user>/<repo name>/tree/<branch name>/<path to content>/"
```

If this is correct I would like to kindly ask if the wiki can be updated to make this easier for the next adopter.

Thanks and best,

tiflor

**Comment by adityatelange** [comment]
Hi @tiflor Yes
https://github.com/adityatelange/hugo-PaperMod/blob/13e69e5662265f5ca6c554b7ab6f82df2e13e85e/config.yml#L157-L160

**Reply by adityatelange** [reply]
Noted, I'll update the docs.

**Reply by tiflor** [reply]
Thanks!
